countrynames/__init__.py

```diff
@@ -36,20 +37,19 @@ def _load_data():
 
 
 def _fuzzy_search(name):
-    matches = []
+    best_code = None
+    best_distance = None
     for cand, code in COUNTRY_NAMES.items():
         if len(cand) <= 4:
             continue
         distance = Levenshtein.distance(name, cand)
-        if cand in name:
-            matches.append((code, distance))
-        elif Levenshtein.distance(name, cand) <= 2:
-            matches.append((code, distance))
-    matches = sorted(matches, key=lambda x: x[1])
-    if len(matches) > 0:
-        match = matches[0][0]
-        log.debug("Guessing country: %s -> %s", name, match)
-        return match
+        if best_distance is None or distance < best_distance:
+            best_distance = distance
+            best_code = code
+    if best_distance > 2:
```

Comment:
I left this in but 2 seems pretty arbitrary. Maybe it should be a percentage of the original string? So instead of saying we'll accept any strings that are 2 edits away, we can accept strings that are 10% away?